sources_test.go

```diff
@@ -254,3 +254,49 @@ func TestStdin(t *testing.T) {
 		t.Errorf("want %q, got %q", want, string(got))
 	}
 }
+
+func TestPrompt(t *testing.T) {
+	tcs := []struct {
+		Name        string
+		Input       string
+		Default     string
+		Want        string
+		ErrExpected bool
+	}{
+		{"User Input OK", "123", "notused", "123", false},
+		{"No User Input", "", "default", "default", false},
+		{"Complex Input", "/user/test @@@ testing one two three", "default", "/user/test @@@ testing one two three", false},
+	}
+	for _, tc := range tcs {
+		t.Run(tc.Name, func(t *testing.T) {
+			want := []byte(tc.Input)
```

Comment:
Actually, the output is going to be a string, isn't it? So the `want` value should be a string too.